common/src/buffers/in_mem/send.rs

```diff
@@ -0,0 +1,233 @@
+use crate::buffers::{DeniablePayload, DenimChunk, Flag, SendingBuffer};
+use crate::denim_message::DeniableMessage;
+use crate::error::LibError;
+use async_trait::async_trait;
+use bincode::config;
+use prost::Message;
+use rand::RngCore;
+use std::collections::VecDeque;
+use std::mem::take;
+
+const DENIM_CHUNK_WITHOUT_PAYLOAD: usize = 4;
+
+const DENIABLE_PAYLOAD_MIN_LENGTH: usize = 20;
```

Comment:
this should be configurable in a configuration and a member of InMemorySendingBuffer